File: app/src/main/java/com/dg/user008/crazyandroid/activity/Demo020503Activity.java
package com.dg.user008.crazyandroid.activity;

import android.app.ListActivity;
import android.os.Bundle;
import android.widget.ArrayAdapter;

public class Demo020503Activity extends ListActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        //setContentView(R.layout.activity_demo020503);
        String[] strings={"hello","world","22222","sssss"};
        ArrayAdapter<String > adapter=new ArrayAdapter<String>(this, android.R.layout.simple_list_item_single_choice,strings);
        setListAdapter(adapter);
    }
}

File: app/src/main/java/com/liushu/crazyandroid/ui/stage01/chapter02/activity/Demo020503Activity.kt
package com.liushu.crazyandroid.ui.stage01.chapter02.activity

import android.app.ListActivity
import android.os.Bundle
import android.widget.ArrayAdapter

class Demo020503Activity : ListActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        //setContentView(R.layout.activity_demo020503);
        val strings = arrayOf("hello", "world", "22222", "sssss")
        val adapter = ArrayAdapter(this, android.R.layout.simple_list_item_single_choice, strings)
        listAdapter = adapter
    }
}
